install/kubernetes/templates/istio-sidecar-injector-configmap-debug.yaml.tmpl

```diff
@@ -0,0 +1,107 @@
+kind: ConfigMap
+metadata:
+  name: istio-inject
+apiVersion: v1
+data:
+  policy: enabled
+  template: |-
+    initContainers:
+    - name: istio-init
+      image: {PILOT_HUB}/proxy_init:{PILOT_TAG}
+      args:
+      - "-p"
+      - {{ .MeshConfig.ProxyListenPort }}
+      - "-u"
+      - 1337
+      imagePullPolicy: IfNotPresent
+      securityContext:
+        capabilities:
+          add:
+          - NET_ADMIN
+        privileged: true
+      restartPolicy: Always
+    - args:
+      - -c
+      - sysctl -w kernel.core_pattern=/etc/istio/proxy/core.%e.%p.%t && ulimit -c
+        unlimited
+      command:
+      - /bin/sh
+      image: alpine
+      imagePullPolicy: IfNotPresent
+      name: enable-core-dump
+      resources: {}
+      securityContext:
+        privileged: true
+    containers:
+    - name: istio-proxy
+      image: {PILOT_HUB}/{PROXY_IMAGE}:{PILOT_TAG}
+      args:
+      - proxy
+      - sidecar
+      - -v
+      - 2
```

Comment:
does this -v 2 still does anything now that we dropped glog ?

Reply:
Not sure. But it should be easy to change in follow-up PR if necessary. I tried to retain the existing injected resource output for now.